odoo/addons/test_translation_import/tests/test_term_count.py

```diff
@@ -319,3 +319,35 @@ def test_export_import(self):
         ])
 
         self.assertEqual(init_translation_count, len(import_translation))
+
+    def test_export_static_templates(self):
+        """ Ensure export+import gives the same result as loading a language """
+        # load language and generate missing terms to create missing empty terms
+        with mute_logger('odoo.addons.base.models.ir_translation'):
+            self.env["base.language.install"].create({'lang': 'fr_FR', 'overwrite': True}).lang_install()
+        self.env["base.update.translations"].create({'lang': 'fr_FR'}).act_update()
+
+        module = self.env.ref('base.module_test_translation_import')
+        export = self.env["base.language.export"].create({
+            'lang': 'fr_FR',
+            'format': 'po',
+            'modules': [(6, 0, [module.id])]
+        })
+        export.act_getfile()
+        po_file = export.data
+        self.assertIsNotNone(po_file)
+
+        import polib
```

Comment:
If you want to avoid working with polib
```python
from odoo.tools.translate import TranslationModuleReader
reader = TranslationModuleReader(self.env.cr, ['test_translation_import'])
for line in reader:
    module, ttype, name, res_id, source, value, comments = line
    if name == "addons/test_translation_import/static/src/xml/js_templates.xml":
      ...
```